src/backend/distributed/sql/citus--11.1-1--11.2-1.sql

```diff
@@ -13,3 +13,12 @@ DROP FUNCTION pg_catalog.worker_append_table_to_shard(text, text, text, integer)
 #include "udfs/citus_task_wait/11.2-1.sql"
 #include "udfs/citus_prepare_pg_upgrade/11.2-1.sql"
 #include "udfs/citus_finish_pg_upgrade/11.2-1.sql"
+
+-- drop orphaned shards after inserting records for them into pg_dist_cleanup
```

Comment:
Should we do anything for downgrades?
What if the user downgrades to a version that `pg_dist_cleanup` doesn't exist?

Reply:
I think the best is to fail the downgrade